README.md

```diff
@@ -9,7 +9,6 @@ tools like [AWS CLI](https://github.com/aws/aws-cli).
 ## Features
 
 * large subset of POSIX including reading/writing files, directories, symlinks, mode, uid/gid, and extended attributes
-* compatible with Amazon S3, Google Cloud Storage, and other S3-based object stores
```

Comment:
Let's just mention Amazon S3 and link to other S3 compatible providers:

https://github.com/s3fs-fuse/s3fs-fuse/wiki/Non-Amazon-S3